src/main/resources/application-construction.yml

```diff
@@ -0,0 +1,9 @@
+# This file defines Logistics pilot configurations for the application.
+# To activate this profile use command 'mvn spring-boot:run -D"spring-boot.run.profiles"=construction'
+application-scenario-config:
+  fclRulesFilePath: rules/ConstructionRobotAssistanceDecisionRules.fcl
+  decisionResultsDescription:
+    AI_AUTONOMOUSLY: "Let the robot continue trying"
+    HUMAN_IN_THE_LOOP: "Warning: inform human so they can decide"
+    HUMAN_ON_THE_LOOP: "Information: inform human so they can decide"
```

Comment:
Please move "on the loop" before "in the loop", that order makes more sense (from highest to lowest AI autonomy). Also update the description text to be in line with suggested README changes.